packages/next/src/server/next-server.ts

```diff
@@ -958,8 +958,15 @@ export default class NextNodeServer extends BaseServer {
     params: Params | null
     isAppPath: boolean
   }): Promise<FindComponentsResult | null> {
-    return getTracer().trace(NextNodeServerSpan.findPageComponents, () =>
-      this.findPageComponentsImpl({ pathname, query, params, isAppPath })
+    return getTracer().trace(
+      NextNodeServerSpan.findPageComponents,
+      {
+        tracerName: `resolving route ${pathname}`,
+        attributes: {
+          'next.route': pathname,
```

Comment:
do we want to pass this for all spans?

Reply:
I'm not sure. It's quite easy to propagate through context, but I'm not sure if there is a need for it.

Maybe we can add it later if we find it useful?

Reply:
I'll do this with providing `http.route`. Guys at sentry say it's needed for moving over to otel.